docs/topics/db/fixtures.txt

```diff
@@ -126,11 +126,34 @@ You could also write a decorator to encapsulate this logic::
 Just be aware that this logic will disable the signals whenever fixtures are
 deserialized, not just during ``loaddata``.
 
-Note that the order in which fixture files are processed is undefined. However,
-all fixture data is installed as a single transaction, so data in
-one fixture can reference data in another fixture. If the database backend
-supports row-level constraints, these constraints will be checked at the
-end of the transaction.
+Fixtures can be loaded in a specified order. For example:
+
+.. code-block:: shell
+
+    manage.py loaddata foo bar baz
+
+or in a test case class:
+
+.. code-block:: python
+
+    class AnimalTestCase(TestCase):
+        fixtures = ["mammals", "birds", "insects"]
+
+In this example, ``mammals`` refers to any fixture named ``mammals`` in any
+application. This would first load all the ``mammals``-named fixtures from all
+applications (in the order in which applications are defined), followed by
+all the ``birds`` and then all the ``insects`` fixtures.
+
+Where there are several fixtures with the same name, the order in which
+applications are defined in ``INSTALLED_APPS`` is also taken into account.
+The application listed first will be the one whose fixture is loaded first.
```

Comment:
I suggest to replace lines 142-149 by the following single sentence, which answers my question:

These examples would first load all the fixtures named ``mammals`` from all applications (in the order in which applications are defined in ``INSTALLED_APPS``), followed by all the ``birds`` and then all the ``insects`` fixtures.

Reply:
I think this is a good proposal. @dtraleigh do you agree? Could you incorporate this?

Reply:
Concise. Yes, I'll add.